cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -38,6 +38,14 @@ Resource = "coredata"
 OutputPath = "accessToken.json"
 JWTFile = "/tmp/edgex/secrets/security-proxy-setup/kong-admin-jwt"
 
+[CORSConfiguration]
+EnableCORS = false
```

Comment:
Don't we want this enabled by default?

Reply:
I would say no.  Enabling CORS allows additional attack surface and to be effective, the CORS domain list needs to be changed from its default to a customer-specific value.

Reply:
Need related documentation PR on how to enable CORS